onnxruntime/core/providers/cpu/tensor/gather_nd.cc

```diff
@@ -125,15 +102,55 @@ Status GatherNDBase::PrepareForCompute(OpKernelContext* context, Prepare& p, con
                         : ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "invalid index found, index = ", err_index);
 }
 
-template Status GatherNDBase::PrepareForCompute<int32_t>(OpKernelContext*, Prepare&, concurrency::ThreadPool*) const;
-template Status GatherNDBase::PrepareForCompute<int64_t>(OpKernelContext*, Prepare&, concurrency::ThreadPool*) const;
+template Status GatherNDBase::PrepareForCompute<int32_t>(const TensorShape& input_shape,
+                                                         const Tensor* indices_tensor,
+                                                         const int64_t bytes_per_value, Prepare& p,
+                                                         concurrency::ThreadPool*) const;
```

Comment:
inconsistent signature -either with or without param names - same for the one below